### ISSUE TITLE
Guj 0x0AC5 shape

### ISSUE THREAD

**Issue by nlci-dev** [nlci/gujr-font-gir#1]
The shape of Gujarati Vowel Sign Candra E is not in the preferred form in the font. 

This is how it currently appears in the font:
![image](https://github.com/nlci/gujr-font-gir/assets/12891019/20828030-e7dc-4401-93bc-4071b5fb6115)

However here's the preferred form:
![image](https://github.com/nlci/gujr-font-gir/assets/12891019/e7ee5d76-f31e-4449-9994-0e7fe439c757)

This font has the correct glyph. 
[GIRR.zip](https://github.com/nlci/gujr-font-gir/files/14622458/GIRR.zip)

**Comment by devosb** [comment]
The characters in the first screenshot do not look like they came from the Gir fonts. The second screenshot looks like the Gir fonts (note how much more variation of stoke width there is in the second screenshot). The form in the green circle can be added to the fonts.

I was surprised by the shape in the green circle, that character does not look like the character in the red circle (and the Unicode code chart looks more like the red circled character). Does that character have different visual forms depending on the style of the font (or for some other reason)?

**Comment by devosb** [comment]
Ah, I see now. This font has 0x0AC5 somewhat like Noto Serif Gujarati, in the Unicode charts this character is more like Noto Sans Gujarati.
![image](https://github.com/nlci/gujr-font-gir/assets/12450072/3ad6ac3c-d444-43cd-bc92-1e1a43fa1b89)

The first line of Gujarati text is the CANDRA E as the independent vowel then the matra on KA. The second line is CANDRA O.

**Comment by nlci-dev** [comment]
You're right, Bobby. We want the Guj Gir font to have notoSerif like shape by default and if possible to have notoSans as a stylistic variant.

**Comment by nlci-dev** [comment]
You're right, @devosb. We want the Guj Gir font to have notoSerif like shape by default and if possible to have notoSans as a stylistic variant.
